<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUi</class>
 <widget class="QDialog" name="LoginUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>辩论开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎使用辩论计时软件 1.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warningButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提示音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="countdownButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>倒计时音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeoutButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>时间到音</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
